netbox/extras/plugins/__init__.py

```diff
@@ -48,10 +48,18 @@ class PluginConfig(AppConfig):
 
     # Default integration paths. Plugin authors can override these to customize the paths to
     # integrated components.
+    template_content_classes = 'template_content.template_content_classes'
```

Comment:
Seems to be some confusion over `template_content` vs `template_content_classes`? I don't have a strong opinion either way, but `template_content` is shorter.